app/web/src/features/dashboard/MyEvents.tsx

```diff
@@ -70,12 +69,9 @@ export default function MyEvents() {
               ))}
           </HorizontalScroller>
           {hasNextPage && (
-            <StyledLink
-              className={classes.allUpcomingEventsLink}
-              to={eventsRoute}
-            >
-              {SHOW_ALL_UPCOMING_EVENTS}
-            </StyledLink>
+            <Button onClick={() => fetchNextPage()}>
+              {SEE_MORE_EVENTS_LABEL}
+            </Button>
```

Comment:
This is a bit weird on mobile (clicking the button makes more appear in the horizontal scroller).
How do you feel about an intersection observer div at the end of the horizontal scroller (only visible at mobile size) which triggers the load more (and displays a spinner?) Like in `InfiniteMessageLoader`.